data/starts.txt

```diff
@@ -59,6 +70,128 @@ conversation "default intro"
+	`	Before long, you arrive in New Reykjavik and find the government office responsible for handling starship loans. The place is relatively empty compared to the photos you'd seen online a week prior; most applicants have already been processed in the week after exam results were posted. The only people left are those sick at the time, those too lazy to apply in a timely manner, and those willing to wait if it means avoiding a mosh pit of students in a government building. You hand over your application papers, and after verifying them against online records, the office worker hands you two cards. One of them is the card for applying for a loan at a nearby bank; the other is your pilot's license, fully paid for.`
+	`	The bank is only a few blocks away from the office. You insert the card into an automatic clerk, and after a soft ping, a large document fills its screen.`
+	`	As you scroll through, a synthesized voice says, "You are borrowing 580,000 credits to be repaid over the course of a year. Your daily interest rate is 0.2%, which means that your daily payments are 2,241 credits, and by the end of the year you will have paid 237,784 credits in interest. Please sign your name on the screen when you are ready."`
+	`	After carefully rereading every page of the loan agreement, you sign your name:`
+	name
+	`	Eager to get started, you rush out of the bank, hop on the metro, and head towards the shipyard. You can barely keep your face straight as the rest of the carriage wear blank expressions. After getting off, you consider throwing away your planetary transport pass, but decide against it.`
+	`	You had seen videos of the shipyard in New Reykjavik, but visiting the site in person blows your wildest dreams out of the water. Throughout the gleaming-white facility, salespeople show of a variety of holographically-rendered vessels to sharply-dressed clients, while skyscraper-sized ships loom silently in the background. Although most of the ships are far outside of your price range, you spot a few that you'll be able to purchase. Which one you choose will determine your future...`
+
+
+
+start "paradise"
+	name "Birthday Surprise"
+	description `You grew up on Mainsail, an oceanic Paradise World terraformed to perfection. Only the richest live on such worlds; luckily, your father is a wealthy financier. But you've always found the rich life boring and are more interested in living a life among the stars. As your eighteenth birthday party winds down, your father takes you aside to talk...`
+	description `This start has has no mortgage, leaving you free to explore the galaxy at your own pace.`
+	thumbnail "scene/mainsail"
+	date 16 11 3013
+	system "Alhena"
+	planet "Mainsail"
+	conversation "paradise intro"
+	account
+		credits 490000
```

Comment:
Small thing, but I feel like a wealthy and cavalier father would just give a round 500,000 credits as a gift rather than a more specific 490,000.
```suggestion
		credits 500000
```